.travis.yml

```diff
@@ -32,17 +33,57 @@ matrix:
       - HDF5_DIR=$HDF5_CACHE_DIR/$HDF5_VERSION
       - H5PY_ENFORCE_COVERAGE=yes
       os: linux-ppc64le
+    - os: linux
+      python: 3.7
+      virt: vm
+      group: edge
+      arch: arm64
+      env:
+        - TOXENV=py37-test-deps
+        - HDF5_VERSION=1.10.5
+        - HDF5_DIR=$HDF5_CACHE_DIR/$HDF5_VERSION
+        - H5PY_ENFORCE_COVERAGE=yes
+        - CIBW_BUILD: cp3[789]-*
+        - CIBW_BEFORE_BUILD: ./ci/get_hdf5_arm64.sh
+        - CIBW_ARCHS: aarch64
 
-before_install:
+  before_install:
   # - export PATH=/usr/lib/ccache:$PATH
   - ccache -s
 
 install:
   - pip install -U tox codecov virtualenv
   - ci/get_hdf5_if_needed.sh
   - ls -lRa $HDF5_CACHE_DIR
+  - if [[ $CIBW_ARCHS == aarch64 ]]; then python -m pip install cibuildwheel==1.9.0; fi
+
 script:
   - tox
+  - if [[ $CIBW_ARCHS == aarch64 ]]; then python3 -m cibuildwheel --output-dir wheelhouse; fi
 
 after_success:
   - python ci/upload_coverage.py
+  # trigger an upload to the shared ecosystem
+  # infrastructure at: https://anaconda.org/scipy-wheels-nightly
+  # for cron jobs only (restricted to master branch once
+  # per week)
+  # SCIPY_WHEELS_NIGHTLY_ACCESS is a secret token
+  # used in Travis CI config, originally
+  # generated at anaconda.org for scipy-wheels-nightly (cron jobs).
+  # For merges (push events) we use the staging area instead;
+  # MULTIBUILD_WHEELS_STAGING_ACCESS is a secret token used in Travis
+  # CI config, originally generated at anaconda.org for
+  # multibuild-wheels-staging
+  #
+  - if [[ $CIBW_ARCHS == aarch64 && "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
+          ANACONDA_ORG="scipy-wheels-nightly";
+          TOKEN=${SCIPY_WHEELS_NIGHTLY_ACCESS};
+    fi;
+    if [[ $CIBW_ARCHS == aarch64 && "$TRAVIS_EVENT_TYPE" == "push" ]]; then
+          ANACONDA_ORG="multibuild-wheels-staging";
+          TOKEN=${MULTIBUILD_WHEELS_STAGING_ACCESS};
+    fi;
```

Comment:
Is this something we're free to use? It looks like only a handful of packages are uploading to either of those orgs frequently. :thinking: 

@ogrisel, I see your picture on both Anaconda organisations. Is it reasonable for h5py's CI to post ARM wheels there, or should we find our own space to do that?

Reply:
I believe that's free, I created an org to test wheel upload for my local testing which was free. https://anaconda.org/test-upload/h5py/files

Reply:
I looked into this, and it appears that anaconda.org has a 3 GB limit per organisation. We're looking at 3 packages somewhere between 4 MB and 10 MB each (I'm not quite sure why the aarch64 wheels you've produced appear to be much bigger than the x86_64 wheels). That seems like we'll fill it up before very long. Although the scipy-wheels-nightly repo seems to have a lot of files in, so maybe that has an exception? :confused: 

The way Azure Pipelines & similar services work is to discard older artifacts after some number of days or number of later CI runs. I wonder if it's reasonable to put files on something like https://transfer.sh/ to achieve something similar, or whether that's abusing a free service.

Reply:
What are your thoughts on using [twine](https://twine.readthedocs.io/en/latest/) for uploading wheels. 
Ref: https://cibuildwheel.readthedocs.io/en/stable/deliver-to-pypi/
I created a test script to upload wheels to [test.pypi](https://test.pypi.org/project/tbbharaj-h5py/3.3.3/#files
) 

https://github.com/tbbharaj/h5py/blob/tbbharaj/upload-wheels/.travis.yml#L85
Builds: https://travis-ci.com/github/tbbharaj/h5py/builds/233562261

Reply:
@takluyver any concerns with using twine?

Reply:
Yes, maybe using test.pypi.org is the way to go. It appears Tensorflow even uploads [`tf-nightly` packages] to the real PyPI, and they're huge (hundreds of MB for each package), so maybe that's OK.

Reply:
Just had a chat with @tacaswell; we'll look again at using the scipy-wheels-nightly repo on anaconda.org.

Reply:
> Just had a chat with @tacaswell; we'll look again at using the scipy-wheels-nightly repo on anaconda.org.

Thanks @takluyver let me know whatever works best, I am happy to make required changes. Thanks again!